libs/hmpb/jest.config.js

```diff
@@ -7,7 +7,7 @@ module.exports = {
   ...shared,
   coverageThreshold: {
     global: {
-      branches: -16,
+      branches: -18,
```

Comment:
The generated fixtures currently don't cover the case that we include customer footer metadata _without_ condensing the metadata. I did try this out manually, and it worked just fine.